app/src/main/java/com/infomaniak/mail/views/AvatarView.kt

```diff
@@ -187,6 +179,32 @@ class AvatarView @JvmOverloads constructor(
         )
     }
 
+    private fun handleDisplayType(
+        avatarDisplayType: AvatarDisplayType,
+        correspondent: Correspondent?,
+        bimi: Bimi?,
+        contacts: MergedContactDictionary,
+    ) {
+        when (avatarDisplayType) {
+            AvatarDisplayType.UNKNOWN_CORRESPONDENT -> loadUnknownUserAvatar()
+            AvatarDisplayType.CUSTOM_AVATAR,
+            AvatarDisplayType.INITIALS -> {
+                loadAvatarUsingDictionary(correspondent!!, contacts)
+                currentCorrespondent = correspondent
+            }
```

Comment:
We should move the `currentCorrespondent = correspondent` inside the loadAvatarUsingDictionary() method like the two other functions